web/cobrands/fixmystreet/staff.js

```diff
@@ -211,6 +211,19 @@ $.extend(fixmystreet.set_up, {
             $select.append($('<option>', { value: kv[0], text: decodeURIComponent(kv[1]) }));
         });
         $select.val(curr_pri);
+        // And the defect types
```

Comment:
Just to reiterate, it would be good if this and the priorities code just above could be shared/act the same, given they are both recreating a select given some data.